examples/message/README.md

```diff
@@ -0,0 +1,21 @@
+# Message - Data Providers Usage Example
```

Comment:
Flatten the directory structure. You probably don't want separate npm package for every example there is - that would be a huge code-repetition. I'd put all examples to just one project (there can be multiple components and multiple routes for these examples)

Reply:
>You probably don't want separate npm package for every example there

That's exactly what I thought we want :) (I was looking for inspiration in some other projects, like https://github.com/reactjs/redux/tree/master/examples)
but ok, I can flatten the directory structure, no problem